src/app.ts

```diff
@@ -170,6 +174,7 @@ class App {
         });
 
         router.get('/api/v1/tee/identity', (req, res, next) => {
+            console.log('request time: ', moment().format())
```

Comment:
Use [winston](https://github.com/winstonjs/winston) to unify and format the whole loging system